### ISSUE TITLE
No crash report when an error occurs in background or popup playback

### ISSUE THREAD

**Issue by floral-qua-floral** [TeamNewPipe/NewPipe#3954]
### Version

- v0.19.7

### Steps to reproduce the bug

Testing this relies upon being able to open a video and know it will fail to load, so for this test I am using #3951, as it causes a crash on every video loaded.
1. Go to a front page tab with videos, or search something.
2. Tap one of the videos. If #3951 is still unpatched it will fail to load and send you to a screen with a crash report.
3. Go back to the previous screen and long-tap a video, then select "Start playing in the background" or "Enqueue in the background".
4. The notification for the player will appear but it won't start playing. A text popup will appear reading "Playing in background" and, a few seconds later, "Could not play this stream". NewPipe will not be sent to a crash screen, so there's no way to access a detailed error report.
5. Close the player notification and long-tap the video again, and select "Start playing in a popup" or "Enqueue in a popup".
6. The popup will open but fail to load the video. A text popup will say "Playing in popup mode", and then a second one a few seconds later will say "Could not play this stream". There will be no way to access an error report for this either.

### Expected behavior

The NewPipe app could be sent to an error screen like when errors occur in the original player. Alternatively, a notification could be sent that the user can tap to bring up an error screen manually.

### Actual behaviour

The text popup cannot be tapped and will fade away quickly. There doesn't seem to be any way to access an error log for the failed playback.

**Comment by Stypox** [comment]
This is afaik intended. Even when everything works good a random video every now and then cannot load: would it be pleasant if NewPipe crashed or interrupted your workflow in that case? Instead, it just tells you that there was an issue. I don't think it is really needed to collect those bug reports, since at the end of the day they would be the exact same bug reports as when opening a video details page directly (so they would be collected anyway), but they would annoy the user much more.

**Comment by wb9688** [comment]
@Stypox: Yes, but sometimes issues don't consistently happen, so this would be useful for those reports. It should however not show an option to show crash reports for e.g. network issues.
